pkg/status/status_server.go

```diff
@@ -236,6 +238,18 @@ func (s *Server) loggersHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) accesslogHandler(w http.ResponseWriter, r *http.Request) {
+	accesslogInfo := r.URL.Query().Get("enable")
+
+	if accesslogInfo == "enable" {
+		s.xdsClient.WorkloadController.MetricController.EnableAccesslog = true
+	} else {
+		s.xdsClient.WorkloadController.MetricController.EnableAccesslog = false
```

Comment:
mutating a var which is being used by another routine can cause data race in golang

Reply:
Add a lock protection for data race